src/server/game/Movement/Spline/MoveSplineFlag.h

```diff
@@ -58,8 +58,8 @@ namespace Movement
 
             // Masks
             Mask_Final_Facing   = Final_Point | Final_Target | Final_Angle,
-            // animation ids stored here, see AnimType enum, used with Animation flag
-            Mask_Animations     = 0xFF,
+            // animation ids stored here, see AimationTier enum, used with Animation flag
+            Mask_Animations     = 0x7,
```

Comment:
This value is supposed to cover all bits specified below for animId, 8 bits means 0xFF

Reply:
Thnx, changed.